constants.go

```diff
@@ -39,6 +39,7 @@ var (
 	SysconfDir        string
 	LocalstateDir     string
 	DbusInterfacesDir string
+	VarLibDir         string
```

Comment:
There's something about this name that doesn't sit right with me. "VarLibDir" makes me think `/var/lib`, not `/var/lib/rhc`. I'd rather we name it something more accurate, like `RhcLocalstateDir`.